.github/workflows/test-release.yaml

```diff
@@ -401,7 +401,7 @@ jobs:
             ~/.cargo/registry
             ~/.cargo/git
             target
-          key: stable-linux-armv7-gnu-node@18-cargo-cache
+          key: stable-linux-armv7-gnueabihf-node@20-cargo-cache
```

Comment:
I didn't add a test for armv7-unknown-linux-muslebihf here because `@napi-rs/cross-toolchain` doesn't support it.